packages/sdk/client/src/packlets/client/utils.ts

```diff
@@ -43,10 +43,14 @@ export const fromIFrame = (
 /**
  * Creates stand-alone services without rpc.
  */
-export const fromHost = (config: Config = new Config()): ClientServicesProvider => {
+export const fromHost = ({
+  config = new Config(),
+  ...params
+}: ClientServicesHostParams = {}): ClientServicesProvider => {
```

Comment:
is this just to forward the lock key?
unless there's a strong need to treat lock keys differently then I'd just derive them from the storage path and not do this change.

Also it makes sense for me to have the lock combined with the storage as that is what it is meant to protect

Reply:
I want to have the lock file and database separately because lock file is runtime data and will be cleared a lot and the database should be persistent

Reply:
Also, I do not want to deal with the issues with browser lock that Josiah described